mails/mail.js

```diff
@@ -0,0 +1,54 @@
+const nodemailer = require("nodemailer");
+
+
+const sendEmail=async function main() {
+  try{
+    // let testAccount = await nodemailer.createTestAccount();
+
+    const transporter = nodemailer.createTransport({
+        host: 'smtp.ethereal.email',
+        port: 587,
+        auth: {
+            user: 'herta.schimmel@ethereal.email',
+            pass: 'pTqUPGtapKHFJFPhkS'
+        },
+        tls: {
+            rejectUnauthorized: false
+        }
+    });
+
+
+    // let transporter = nodemailer.createTransport({
+    //   host: "smtp.ethereal.email",
+    //   port: 587,
+    //   secure: false, // true for 465, false for other ports
+    //   auth: {
+    //     user: testAccount.user, 
+    //     pass: testAccount.pass, 
+    //   },
+    // });
+  
+    
+    let info = await transporter.sendMail({
+      from: '"Fred Foo 👻" <herta.schimmel@ethereal.email>', 
+      to: "sai2aeronautical@gmail.com", 
```

Comment:
Once everything is working 

Keep the to address to this mail techswap.iit@gmail.com